hrui/tests/phpunit/bootstrap.php

```diff
@@ -4,6 +4,14 @@
 ini_set('safe_mode', 0);
 eval(cv('php:boot --level=classloader', 'phpcode'));
 
+// Add other extensions to include path to allow use of CRM/* classes
+$civiHRRoot = realpath(__DIR__ . '/../../../');
+$extensionDirs = glob($civiHRRoot . '/*', GLOB_ONLYDIR);
+foreach ($extensionDirs as $extensionDir) {
+  $include_path = $extensionDir . PATH_SEPARATOR . get_include_path( );
```

Comment:
I've tried installing hrcore as you suggested in chat but ran into more errors regarding failed validation on custom CSS URL, errors in raw SQL queries in upgraders and generally a whole lot of stuff that has nothing at all to do with this test. This is pretty frustrating, how an unrelated change in an installer will break this a test in this extension. I'd prefer to avoid trying to fix every problem with the other extensions in this ticket, so unless you have an alternative maybe we can leave it like this.